docs/database-schema.md

```diff
@@ -0,0 +1,13 @@
+# Database Schema
+
+### wp_notifications table
+* id (int) 11 auto_increment `ID of the notification.`
+* sender `Sender of the notification.`
+* timestamp `Timestamp of when the notification was triggered.`
+* recipients `Collection of notification recipients.`
+* title `Notification message.` (optional)
+* message `Notification message.`
+* action_link `Correctly formatted URL to an internal or external action.`
+* status enum (READ, UNREAD) `Notification status.`
```

Comment:
Beware that the `status` needs to be tied to each recipient. This leaves two possibilities:

1. The `recipients` field become `recipient` to only contain one recipient, and the notifications are duplicated for each recipient.
2. We create a relations table (let's name it `wp_sent_to`) which maps recipients to notification AND stores the `status` for each recipient-notification relation.

Reply:
@raaaahman if we follow database normalization guidelines, we should probably create a separate table. However, I am concerned about the possible pushback from Core in creating a new table as part of (eventually) merging this feature. Are there any downsides with going with option 1?